### PR TITLE
ci(upgrades): change initial cluster version

### PR DESCRIPTION
**Pull request by sreis**
* Use the latest available release from https://go.otr.dev/cli-latest-release-linux as the initial cluster.
* Fetch CLI artifacts hosted by HTTP servers.

### CODE REVIEW

**Comment by jgehrcke** [comment]
oh that is funny :D I opened this one in basically the same minute as you did: https://github.com/opstrace/opstrace/pull/1081 -- well. You decide, feel free to close my PR :).

File: ci/test-upgrade/fetch-cli-artifacts.sh [hunk 1]
```diff
@@ -32,11 +44,18 @@ fetch_cli_artifact() {
 
     mkdir -p ${dir}
 
-    if [ -f "${artifact}" ]; then
-        cp ${artifact} ${dir}
-    else
-        fetch_cli_s3_artifact ${artifact} ${dir}
-    fi
+    case "${artifact}" in
```
Comment (jgehrcke) [inline ci/test-upgrade/fetch-cli-artifacts.sh:47]: `case "${artifact}" in` oh is that a magic string prefix check? :D

Comment (sreis) [reply ci/test-upgrade/fetch-cli-artifacts.sh:47]: Yep. Bash has some nice features [built in](https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_03.html).

Comment (jgehrcke) [reply ci/test-upgrade/fetch-cli-artifacts.sh:47]: Oh I see, this is generic pattern matching!

**Comment by sreis** [comment]
> oh that is funny :D I opened this one in basically the same minute as you did: #1081 -- well. You decide, feel free to close my PR :).

Ups... I thought I was supposed to take care of this after our conversation in slack :sweat_smile:

**Pull request merged**